src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.tsx

```diff
@@ -5,11 +5,10 @@ import Button, { ButtonProps } from "../../inline/Button";
 import s from "./TwoParagraphBlock.module.css";
 
 /* Main component */
-
 type TwoParagraphBlockProps = {
   title: string;
   paragraph1: string;
```

Comment:
Ah, in that case, then we should probably make `paragraph1` a `ReactNode` as well, for consistency.